CHANGELOG.md

```diff
@@ -22,6 +22,11 @@ and this project adheres to
 
 ### Changed
 
+- [#5118](https://github.com/firecracker-microvm/firecracker/pull/5118): Cleared
+  WAITPKG CPUID bit in CPUID normalization since it may lead to an overhead to
```

Comment:
what CPUs are affected?

Reply:
Sapphire Rapids

Reply:
yeah maybe just note it in the changelog: "these insns are available in SPR+"

Reply:
I would also reword as: "which enable the guest to force a host CPU into an idle state, which is undesirable in a FaaS environment, as we want the host to make that decision".